articles/active-directory/develop/msal-net-xamarin-android-considerations.md

```diff
@@ -107,6 +106,102 @@ Here's an example of a class that represents the values of the XML file:
   }
 ```
 
+### System Browser Support in Brokered Authentication
```

Comment:
```suggestion
### System browser support in brokered authentication

```

* Sentence-style casing in all headings
* Newline after all headings (linter check)
* Consider rewording this heading to be more intent-based (see next comment)